lib/ansible/modules/cloud/ovirt/ovirt_host_networks.py

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/python
```

Comment:
in your commit message, explain why this change is useful (lets users control fine details of Linux bonding)

Reply:
And add some useful examples.